### PR TITLE
Indigo Disk sprites

### PR DESCRIPTION
**Pull request by AlexOn1ine**
Partial fix for #3881 

pokeemerald-29
pokeemerald-28
pokeemerald-27
pokeemerald-26
pokeemerald-32
pokeemerald-31

### CODE REVIEW

**Comment by AsparagusEduardo** [comment]
Tip for the reviewer: check for the shiny palettes

**Comment by AlexOn1ine** [comment]
> Tip for the reviewer: check for the shiny palettes

pokeemerald-33
pokeemerald-34
pokeemerald-35
pokeemerald-36
pokeemerald-37
pokeemerald-38

**Comment by AsparagusEduardo** [comment]
I guess I'm the reviewer now xD

File: src/data/pokemon/species_info/gen_8.h [hunk 1]
```diff
@@ -4333,15 +4333,15 @@ const struct SpeciesInfo gSpeciesInfoGen8[] =
...
-        .backPicYOffset = 0,
-        //.backAnimId = BACK_ANIM_NONE,
-        //PALETTES(Archaludon),
-        //ICON(Archaludon, 0),
+        .frontAnimId = ANIM_V_SQUISH_AND_BOUNCE,
+        BACK_PIC(Archaludon, 64, 64),
+        .backPicYOffset = 7,
+        .backAnimId = BACK_ANIM_NONE,
+        PALETTES(Archaludon),
+        // ICON(Archaludon, 0),
```
Comment (AsparagusEduardo) [inline src/data/pokemon/species_info/gen_8.h:4344]:
```suggestion
        //ICON(Archaludon, 0),
```

File: src/data/pokemon/species_info/gen_9.h [hunk 2]
```diff
@@ -5468,15 +5468,15 @@ const struct SpeciesInfo gSpeciesInfoGen9[] =
...
-        .backPicYOffset = 0,
-        //.backAnimId = BACK_ANIM_NONE,
-        //PALETTES(GougingFire),
-        //ICON(GougingFire, 0),
+        .frontAnimId = ANIM_V_SQUISH_AND_BOUNCE,
+        BACK_PIC(GougingFire, 64, 64),
+        .backPicYOffset = 6,
+        .backAnimId = BACK_ANIM_NONE,
+        PALETTES(GougingFire),
+        // ICON(GougingFire, 0),
```
Comment (AsparagusEduardo) [inline src/data/pokemon/species_info/gen_9.h:5479]:
```suggestion
        //ICON(GougingFire, 0),
```

File: src/data/pokemon/species_info/gen_9.h [hunk 3]
```diff
@@ -5518,15 +5518,15 @@ const struct SpeciesInfo gSpeciesInfoGen9[] =
...
-        //BACK_PIC(RagingBolt, 64, 64),
+        .frontAnimId = ANIM_V_SQUISH_AND_BOUNCE,
+        BACK_PIC(RagingBolt, 64, 64),
         .backPicYOffset = 0,
-        //.backAnimId = BACK_ANIM_NONE,
-        //PALETTES(RagingBolt),
-        //ICON(RagingBolt, 0),
+        .backAnimId = BACK_ANIM_NONE,
+        PALETTES(RagingBolt),
+        // ICON(RagingBolt, 0),
```
Comment (AsparagusEduardo) [inline src/data/pokemon/species_info/gen_9.h:5529]:
```suggestion
        //ICON(RagingBolt, 0),
```

File: src/data/pokemon/species_info/gen_9.h [hunk 4]
```diff
@@ -5567,15 +5567,15 @@ const struct SpeciesInfo gSpeciesInfoGen9[] =
...
-        .backPicYOffset = 0,
-        //.backAnimId = BACK_ANIM_NONE,
-        //PALETTES(IronBoulder),
-        //ICON(IronBoulder, 0),
+        .frontAnimId = ANIM_V_SQUISH_AND_BOUNCE,
+        BACK_PIC(IronBoulder, 64, 64),
+        .backPicYOffset = 12,
+        .backAnimId = BACK_ANIM_NONE,
+        PALETTES(IronBoulder),
+        // ICON(IronBoulder, 0),
```
Comment (AsparagusEduardo) [inline src/data/pokemon/species_info/gen_9.h:5578]:
```suggestion
        //ICON(IronBoulder, 0),
```

File: src/data/pokemon/species_info/gen_9.h [hunk 5]
```diff
@@ -5617,15 +5617,15 @@ const struct SpeciesInfo gSpeciesInfoGen9[] =
...
-        .backPicYOffset = 0,
-        //.backAnimId = BACK_ANIM_NONE,
-        //PALETTES(IronCrown),
-        //ICON(IronCrown, 0),
+        .frontAnimId = ANIM_V_SQUISH_AND_BOUNCE,
+        BACK_PIC(IronCrown, 64, 64),
+        .backPicYOffset = 2,
+        .backAnimId = BACK_ANIM_NONE,
+        PALETTES(IronCrown),
+        // ICON(IronCrown, 0),
```
Comment (AsparagusEduardo) [inline src/data/pokemon/species_info/gen_9.h:5628]:
```suggestion
        //ICON(IronCrown, 0),
```

**Review by AsparagusEduardo — CHANGES_REQUESTED** [review]

**Review by AsparagusEduardo — APPROVED** [review]

**Pull request merged**